dialects/paulistes.json

```diff
@@ -200,6 +200,15 @@
       "Descolei um emprego novo, começarei amanhã",
       "Descola aquela grana para mim"
       ]
+  },
+  {
+    "dialect": "MC (Como em MC Donalds)",
```

Comment:
Hi, first, thanks for contributing :]

The phrase in parentheses `(Como em MC Donalds)` is it part of the dialect?

Reply:
What do you think of using synonyms? 

`"synonyms": "MC Donalds"`

Reply:
I thought about explaining as a meaning, it would look like:
"meanings": "Deve ser lido como o 'Mc' em "McDonalds"